Web.md

```diff
@@ -123,6 +123,48 @@ MIME type mismatch or `opaque` response types
 [reject](http://tc39.github.io/ecma262/#sec-rejectpromise) the Promise with a
 `WebAssembly.CompileError`.
 
+## Developer-facing display conventions
+
+Browsers, JavaScript engines, and offline tools have common ways of referring to
+JavaScript artifacts and language constructs. For example, locations in
+JavaScript source code are printed in stack traces or error messages, and are
+represented naturally as decimal-format lines and columns in text files. Names
+of functions and variables are taken directly from the sources. Therefore (for
+example) even though the exact format of Error.stack strings does not always
+match, the locations are easily understandable and the same across browsers.
+
+To achive the same goal of a common representations for WebAssembly constructs, the
+following conventions are adopted.
+
+A wasm location is a reference to a particular instruction in the binary, and may be
+displayed by a browser or engine in similar contexts as JavaScript source locations.
+It has the following format:
+`${url}:wasm-function[${funcIndex}]:0x${pcOffset}`
+Where
+* `${url}` is the URL associated with the module (e.g. via a response object), if any.
+* `${funcIndex}` is an index the [function index space](https://github.com/WebAssembly/design/blob/master/Modules.md#function-index-space).
+* `${pcOffset}` is the offset in the module binary of the first byte of the instruction, printed in hexadecimal with lower-case digits.
```

Comment:
Yes, but in this formulation the `0x` is part of the template and not part of the substituted value. Do you think we should switch that? It would mean that this line would say something like "`${pcOffset}` is the offset ... printed in hexadecimal with lower-case digits and a leading `0x` prefix" which seems a little more awkward, but I don't have a strong opinion on that.